requirements.txt

```diff
@@ -3,3 +3,4 @@ pytest==7.4.3
 pytest-cov==5.0.0
 pytest-mock==3.14.0
 PyGithub==1.59.0
+pylint~=3.2.6
```

Comment:
Extra line here.

Reply:
Just a quick answer. Do you want to have exact these versions of dependecies? `pytest-cov==5.0.0` if it is not the cause, I would recommended to write requirements with ~ instead, like you have in pylint.

Reply:
Thanks for pointing out it. Here is the risk in the future for the not-tested combination. What do you think?

Reply:
I understand the risk, if we want to avoid the risk, then we should probably keep the `==` in the all requirements. Right?

Reply:
Yeah, and from time to time, do version upgrades supported by testing.

Reply:
So, please change pylint and requests signs into `==`.

Reply:
Done.